.prettierrc

```diff
@@ -0,0 +1,13 @@
+{
+  "trailingComma": "es5",
+  "tabWidth": 2,
+  "semi": true,
+  "singleQuote": true,
+  "bracketSpacing": false,
+  "printWidth": 160,
+  "overrides": [
```

Comment:
I think it's not relevant in our case. Can you please try to format the code without this param?

Reply:
There is four params in code snippet. What exact param do you mean?